content/ja/account_management/rbac/permissions.md

```diff
@@ -22,7 +22,7 @@ title: Datadog ロールのアクセス許可
 
 ### すぐに使えるロール
 
-By default, existing users are associated with one of the three out-of-the-box roles:
+By default, existing users are associated with one of the three managed roles:
```

Comment:
i'm not sure what the process is for getting equivalent changes made to this translated section

Reply:
We have a separate process for translating docs, any changes outside of that process is usually overwritten. No action needed, but something to be aware of if you notice the Japanese docs revert back to `out-of-the-box`.